packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -374,4 +357,23 @@ export default class Run extends SfdxCommand {
     const hint = messages.getMessage('apexTestReportFormatHint', [reportArgs]);
     return hint;
   }
+
+  /** Handles special exceptions where we don't want to return early
+   * with the testRunId.*/
+  private shouldExitEarly(): boolean {
+    if (this.flags.resultformat !== undefined) {
+      return false;
+    }
+
+    //W-9346875
```

Comment:
nit I'd prefer real deal comments here and on 373

Reply:
Real deal meaning /**/ with more context?

Reply:
My $0.02: include the wrk ID, but also provide some details & context.

Reply:
more context.  We are OSS after all. These are meaningless to anyone that can't get to GUS.  My personal preference is to not include ticket info in code comments.  Git is _really_ good at tracking this change back to a commit/PR.  This is the way.